src/internal/modules/contact-entry/forms/company-search-select-company.js

```diff
@@ -7,12 +7,13 @@ const getCompanyText = row => {
 };
 
 const getCompanyChoices = companies => {
-  const choices = companies.map(record => {
-    return ({
-      value: record.company.companyNumber,
-      label: getCompanyText(record)
-    });
-  });
+  const choices = [{ value: null, label: `${companies.length} companies found` },
```

Comment:
Does the first choice with `null` value here get rendered with a radio button?

Reply:
Good question.
It's a dropdown menu rather than radio, and the null is intended to ensure that the validation kicks in, so that you can't accidentally select that first <option> which is intended to display the result count.

![Screenshot 2020-10-19 at 13 53 11](https://user-images.githubusercontent.com/6029895/96453496-77969280-1212-11eb-8c3b-60b73de40f02.png)